modules/platforms/dotnet/Apache.Ignite.Core/Cache/Configuration/CacheConfiguration.cs

```diff
@@ -948,5 +957,11 @@ public string MemoryPolicyName
         /// </summary>
         [IgniteExperimental]
         public PlatformCacheConfiguration PlatformCacheConfiguration { get; set; }
+
+        /// <summary>
+        /// Gets or sets node attribute filters configuration.
+        /// More details: <see cref="AttributeNodeFilter"/>. 
+        /// </summary>
+        public AttributeNodeFilter NodeFilter { get; set; }
```

Comment:
Let's leave a room for improvement here and use `IClusterNodeFilter` interface (already exists).
Mention that only a predefined `AttributeNodeFilter` implementation is supported here.

Reply:
On the second thought - we already have `PlatformClusterNodeFilterImpl` in Java that is used for services, can we just use that for caches as well?